crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst

```diff
@@ -9,6 +9,43 @@ Changelog - nrf_cc3xx_mbedcrypto
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_mbedcrypto - 0.9.16
+*****************************
+
+New version of the runtime library with the following bug fixes and improvements:
+
+* Built with MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER enabled to ensure support with and without TF-M.
+* Added support for zero input message length for EDDSA for RFC test compliance.
+* Removed unused trace functions from runtime library.
+
+Library built against Mbed TLS version 3.1.0.
+
```

Comment:
```suggestion
```

This is not a change.

Reply:
This is not a change statement

Reply:
The changelog-items wil be handled in a subsequent PR. We must prioritize adding this new library. 

@SebastianBoe: Please add  a JIRA-ref in the comment and resolve, so we can get this in